assets/js/modules/pagespeed-insights/datastore/constants.js

```diff
@@ -19,11 +19,13 @@
 export const STORE_NAME = 'modules/pagespeed-insights';
 export { STORE_NAME as MODULES_PAGESPEED_INSIGHTS };
 
-// Form ID for PageSpeed widget.
-export const FORM_DASH_WIDGET = 'pagespeedDashboardWidget';
 // Report strategies.
 export const STRATEGY_MOBILE = 'mobile';
 export const STRATEGY_DESKTOP = 'desktop';
 // Report data sources.
 export const DATA_SRC_LAB = 'data_lab';
 export const DATA_SRC_FIELD = 'data_field';
+
+// Dashboard widgets using the UI store.
+export const WIDGET_STRATEGY = 'dashboardPageSpeedWidgetStrategy';
+export const WIDGET_DATA_SOURCE = 'dashboardPageSpeedWidgetDataSource';
```

Comment:
Let's use a consistent naming format for these key variables across the plugin. I like to keep the constant names short, but maybe we should prefix them to indicate that they are fields for the `core/ui` store. The alternative would be to go with super-verbose names (like what is used for the actual values of these constants), but since these constants are usually not mixed between different components, it should be okay keeping their names short.

How about renaming these to `UI_STRATEGY` and `UI_DATA_SOURCE`?

Reply:
Sounds good. Updated @felixarntz